web/source/keymanweb.js

```diff
@@ -3060,8 +3072,7 @@
   **/
   keymanweb.setDfltKeyboard=function()
   {
-    keymanweb._SetActiveKeyboard('Keyboard_us','eng',true);
-    keymanweb.doKeyboardChange('Keyboard_us','eng',true);
+    keymanweb.setActiveKeyboard(''); // This line is used in all the UI elements to set the default keyboard, so we should follow suit here.
```

Comment:
The current implementation of `setDfltKeyboard` will fail to complete its intuitive objective, as the `_SetActiveKeyboard` function doesn't have any special handling for 'Keyboard_us' as a default keyboard.  It looks for the empty string `''` instead, so we give that here now.

By calling `keymanweb.setActiveKeyboard('')` instead of the private version, we ensure that any necessary system calls occur.  In particular, `keymanweb.doKeyboardChange` is called there in an appropriate manner.